ansible/configs/kni-osp/default_vars.yml

```diff
@@ -0,0 +1,686 @@
+################################################################################
+################################################################################
+### Environment Structure
+################################################################################
+################################################################################
+
+################
+### Env vars ###
+################
+#CHANGEME
+guid: kniosp                            # Your Global UNIQUE Identifier
+
+env_type: kni-osp          # Name of config to deploy
+cloud_provider: osp                     # Which AgnosticD Cloud Provider to use
+
+output_dir: /tmp/output_dir                # Writable working scratch directory
+email: name@example.com                   # User info for notifications
+
+ansible_user: cloud-user
+remote_user: cloud-user
+
+chomped_zone_internal_dns: "students.osp.opentlc.com"
+
+# The domain that you want to add DNS entries to
+osp_cluster_dns_zone: students.osp.opentlc.com
+
+# The dynamic DNS server you will add entries to.
+# NOTE: This is only applicable when {{ use_dynamic_dns}} is true
+osp_cluster_dns_server: ddns01.opentlc.com
+
+repo_method: satellite
+
+set_repositories_satellite_hostname: labsat.opentlc.com
+set_repositories_satellite_activationkey: kni_lab_key
+set_repositories_satellite_org: Red_Hat_GPTE_Labs
+
+install_student_user: true
+student_password: Redhat01
+
+create_unused_security_groups: False
+
+update_all_packages: false
```

Comment:
`update_packages` is the var that the `common` role uses. Remove this if not needed